automatic/jenkins/jenkins.nuspec

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+    <metadata>
+        <id>jenkins</id>
+        <version>2.222.4</version>
+        <packageSourceUrl>https://github.com/git@github.com:chocolatey-community/chocolatey-packages/tree/master/automatic/jenkins</packageSourceUrl>
```

Comment:
I can tell what you are using as the source when pulling/pushing from the repository 😆 

No need to change anything, it should be automatically updated as part of the run of `update.ps1`